app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const app = express();
+const { getAllTopics } = require("./controllers/topics-controller");
+const { handleServerErrors } = require("./errors/errors");
+
+app.use(express.json());
```

Comment:
We don't need this middleware at this point. Please remove the code which is not related to this ticket. You can always bring it back when you need it